### PR TITLE
MF-1580 - Influxdb Writer changes format of update-time to string

### LINKED ISSUE #1580: Influxdb Writer changes format of update-time to string
**BUG REPORT**

1. What were you trying to achieve?
I am using influxdb to store telemetry data, and while sending update-time as float data type to mainflux mqtt-broker, I noticed that the influxdb-writer service converts the update-time's data type from float to string. The writer shouldn't convert the datatype of incoming data and keep it as it is.

3. What are the expected results?
I expect update-time to be a floating-point number.

4. What are the received results?
Update time is stored as a string.

5. What are the steps to reproduce the issue?
Run latest Docker composition, run InfluxDB Writer, provision and connect thing and channel, send message to any of adapters.

6. In what environment did you encounter the issue?
Latest Docker compose; tested with multiple systems.

7. Additional information you deem important:
N/A

### PR DESCRIPTION
**Pull request by Hasan98-git**
### What does this do?
This will stop influxdb-writer service from changing the datatype of update-time variable to string

### Which issue(s) does this PR fix/relate to?
Resolves #1580

### List any changes that modify/break current functionality
It won't break current functionality

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### CODE REVIEW

File: consumers/writers/influxdb/fields.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 type fields map[string]interface{}
 
 func senmlFields(msg senml.Message) fields {
-	updateTime := strconv.FormatFloat(msg.UpdateTime, 'f', -1, 64)
+	// updateTime := strconv.FormatFloat(msg.UpdateTime, 'f', -1, 64)
```
Comment (manuio) [inline consumers/writers/influxdb/fields.go]: You can simply remove it

**Review by manuio — DISMISSED** [review]

File: consumers/writers/influxdb/fields.go (hunk 1 above)
Comment (Hasan98-git) [reply consumers/writers/influxdb/fields.go]: Okay, but I have checked that the build fails, I thing i will need to remove the strconv package also and replace retur
n value of updateTime with msg.UpdateTime

**Comment by dborovcanin** [comment]
@Hasan98-git Please update tests accordingly.

**Comment by Hasan98-git** [comment]
@dusanb94 can you guide on how and where to update the tests?

**Comment by dborovcanin** [comment]
> @dusanb94 can you guide on how and where to update the tests?

Sure, @Hasan98-git. You can exec tests locally via `go test --race -v ./... -count=1` (you have to have Docker service running). You'll notice that error is in the InfluxDB reader. You can run the same command to form `readers/influxdb` to check what's the probelem.

**Comment by dborovcanin** [comment]
Hello, @Hasan98-git. Tests fix is a little complex and out of the scope of this PR. We should probably switch to Flux and InfluxDB v2 anyways, but that's also not in the scope of this PR. Would you allow me to push test fixes (it's actually Reader update) to this branch to prevent breaking the `master` branch? I want to keep your contribution, but I'd also like to keep the latest codebase working.

**Review by dborovcanin — APPROVED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Comment by Hasan98-git** [comment]
@dusanb94 I have worked on the test but I wasn't able to figure out why it was failing, the issue is that after removing the strconv functionality from consumers/writers/influxdb/fields.go the writer writes the vlaue of updateTime as a zero e-g in the test (readers/influxdb/messages_test.go) the value of updateTime was set to 1234 but the writer writes it as 0 in the db due to which the test fails, can you help?

**Comment by Hasan98-git** [comment]
@dusanb94 Hi, yeah that will be great, thanks for the help.

**Pull request merged**